src/components/Transaction.js

```diff
@@ -1,12 +1,13 @@
 import React from 'react'
 
-const Transaction = () => {
+
+const Transaction = (tx, i) => {
   return (
-    <tr>
-      <td>{"...your code here"}</td>
-      <td>{"...your code here"}</td>
-      <td>{"...your code here"}</td>
-      <td>{"...your code here"}</td>
+    <tr key={i}>
```

Comment:
You don't need to put the key here - that only happens when you're creating an array of items. Also, what you're calling `tx` is actually `props`. Think of this function as a component